netbox/project-static/src/htmx.ts

```diff
@@ -1,11 +1,12 @@
-import { getElements, isTruthy } from './util';
 import { initButtons } from './buttons';
+import { initClipboard } from './clipboard'
 import { initSelects } from './select';
 import { initObjectSelector } from './objectSelector';
 import { initBootstrap } from './bs';
+import { initMessages } from './messages';
 
 function initDepedencies(): void {
-  for (const init of [initButtons, initSelects, initObjectSelector, initBootstrap]) {
+  for (const init of [initButtons, initClipboard, initSelects, initObjectSelector, initBootstrap, initMessages]) {
```

Comment:
is initClipboard and initMessages required?  Wasn't obvious they were part of this PR.